packages/govuk-frontend/src/govuk/helpers/_focused.scss

```
@@ -26,3 +26,24 @@
   // box-shadow is applied to each fragment independently.
   box-decoration-break: clone;
 }
+
+/// Focused image
+///
+/// Provides an outline to clearly indicate when the target element is focused.
+/// Used for non-text content contained within links.
+///
+/// @access public
+
+@mixin govuk-focused-image {
+  outline: $govuk-focus-width solid transparent;
+  background-color: $govuk-focus-colour;
+  box-shadow:
+    -4px -4px $govuk-focus-colour,
+    4px -4px $govuk-focus-colour,
+    4px 4px $govuk-focus-colour,
+    -4px 4px $govuk-focus-colour,
+    -8px -8px $govuk-focus-text-colour,
+    8px -8px $govuk-focus-text-colour,
+    8px 8px $govuk-focus-text-colour,
+    -8px 8px $govuk-focus-text-colour;
```

I think we can get away with just using spread here:

```suggestion
  box-shadow:
    0 0 0 4px $govuk-focus-colour,
    0 0 0 8px $govuk-focus-text-colour;
```

---

🤦 I thought of that, but for some reason thought that spread was only supported on `text-shadow`. I'm blaming that Friday 5pm brain.